nucypher/blockchain/eth/sol/source/contracts/StakingEscrow.sol

```diff
@@ -44,11 +45,13 @@ interface WorkLockInterface {
 /**
 * @title StakingEscrowStub
 * @notice Stub is used to deploy main StakingEscrow after all other contract and make some variables immutable
+* @dev |v1.0.0|
 */
 contract StakingEscrowStub is Upgradeable {
     using AdditionalMath for uint32;
 
     NuCypherToken public immutable token;
+    uint32 public immutable formerSecondsPerPeriod;
```

Comment:
```suggestion
    uint32 public immutable genesisSecondsPerPeriod;
```
I understand that the retention of this parameter is to help the conversion and migration. However, it's quite possible that post-L2 upgrade, there will be good reason to change the period duration again – e.g. to accommodate users who need very short-living policies. So to distinguish the various period durations over the long-term, I think we should give each one a unique label.

Reply:
✔️